vault/dendron.release.2021-11-09.md

```diff
@@ -0,0 +1,92 @@
+---
+id: EjPEcAkZIDgzHaVMk4teA
+title: Release Notes (version 0.68)
+desc: ''
+updated: 1636484692932
+created: 1636478484110
+---
+
+Dendron 0.68 has sprouted  🌱
+
+Take a look at the latest [Dendron wiki](https://wiki.dendron.so), and you'll see some new features directly from these highlights:
+
+* Local TOC in the right-hand nav for each page in the published docs
+* Newly combined search bar that can go directly to a note, or search note contents using the `?` prefix
+
+`Dendron: Show Preview` used to add some extra latency when rendering the preview and also made features like copy/paste unstable. This change migrates the old iframe-based preview to be a native webview. It brings the following changes:
+
+* Faster webview performance
+* Clipboard support (copy and paste works)!
+* Live color theme updates (vs reloading your window for changes to take effect)
+* Bundle native preview with webpack
+* Build native preview on `yarn setup`
+* Cleanup of assets 
+
+### Highlights
+* feat(schemas): adding new command - create schema from hierarchy ([[docs|dendron.topic.schema.create-from-hierarchy]])
+* feat(views): native dendron preview
+* feat(publish): add local table of contents, of current note, in right-hand nav ([[docs|dendron.topic.publish.features#table-of-contents]])
+* feat(publish): merge lookup and search fields ([[docs|dendron.topic.publish.features#lookup-and-search]])
+
+### Everything Else
+* enhance(schemas): adding error messaging and additional validation for when schemas are malformed
+* enhance(schemas): cmd create-schema-from-hierarchy allow schema to end with `*`
+* enhance(schema): update schemas on save instead of having to reload workspace
+* enhance(commands): move header supports multi vault
+* enhance(commands): allow creation of new notes when move header destination doesn't exist yet
+* fix(pods): github import pod handle deleted authors
+* fix(workspace): warning to remove stub property if the note has content
+* fix(publish): table of contents layout
+* fix(server): improving error response on api server
+* fix(workspace): Change Workspace command recognizes native workspaces
+
+#### Breaking Changes
+* enhance(publish): migrate publish related commands to nextjs @hikchoi
+
+Transitioning site build and previewing commands to use nextjs (instead of 11ty), along with the renaming of commands to be more inline with CLI:
+* `Dendron: Site Preview` is renamed to `Dendron: Publish Dev`
+* `Dendron: Site Build` is renamed to `Dendron: Publish Export`
+
+### Community
+
+#### General Updates
+
+- Join the discussion around the [Dendron Site Reorganization](https://github.com/dendronhq/dendron/discussions/1665), where we're refactoring the hierarchies for a better user experience
```

Comment:
might be a bit late to join the discussion. would frame it more as a notification to let people know that we're re-organizing this week with the option to add feedback